src/scripts/main.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const rows = [...tbody.querySelectorAll('tr')];
```

Comment:
You are currently querying all rows in the table at the start of the script and storing them in a variable. However, this list will not be updated when you add or remove rows in the table. This can lead to unexpected behavior when you try to iterate over the rows, for example in the 'appendColumn' and 'removeColumn' event listeners. Instead of storing the rows in a variable, you should query them inside the event listeners to always get the current list of rows.